dask/dataframe/backends.py

```diff
@@ -45,6 +49,12 @@
 from dask.sizeof import SimpleSizeof, sizeof
 from dask.utils import is_arraylike, is_series_like, typename
 
+
+def set_backend(df_backend):
+    """Set the default DataFrame backend to use for input IO"""
+    return config.set({"dataframe.backend": df_backend})
+
+
```

Comment:
This function doesn't appear to be used anywhere (probably left over from a previous commit)

Reply:
Yeah, I agree this utility isn't used/tested, and probably isn't necessary